src/tree-view/styles.scss

```diff
@@ -0,0 +1,49 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use '../internal/styles/tokens' as awsui;
+@use '../internal/styles' as styles;
+
+.root {
+  @include styles.styles-reset;
+
+  font-family: awsui.$font-family-base;
+  font-size: awsui.$font-size-body-m;
+  line-height: awsui.$line-height-heading-m;
+}
+
+.tree,
+.treeitem-group,
+.treeitem {
+  @include styles.styles-reset-ul;
+}
+
+.treeitem {
+  display: grid;
+  grid-template-columns: 28px 1fr;
```

Comment:
nit: let's move the values like 28px or 32px to the named vars in the file's top, for readability and better maintainability, should we use those in multiple places, which we might as well need once we start adding support for connector lines.